base/docs/helpdb/Base.jl

```diff
@@ -7601,8 +7601,8 @@ procs(::SharedArray)
 """
     mod(x, y)
 
-Modulus after flooring division, returning in the range \[0,`y`), if `y` is positive, or
-(`y`,0\] if `y` is negative.
+Modulus after flooring division, returning in the range `[0,y)`, if `y` is positive, or
+`(y,0]` if `y` is negative.
```

Comment:
same applies to these ranges, this isn't julia syntax so use latex math style double backticks